src/App.css

```diff
@@ -1,6 +1,7 @@
 body{
   margin:0;
-  background-color: #FAF9F6;
+  background-color: #fef8fd;
+  /* color:#f7edf2; */
```

Comment:
This code commented will be used? If not, you can remove it

Reply:
Just did. Thanks Vic!